From: Pramod Kumar <pramod@chelsio.com>
Date: Fri, 21 Nov 2014 09:36:35 -0600
Patch-mainline: v3.19-rc1
Git-commit: 63a71ba6178a1989dcea1c5f595b6c6a3d48d6d9
References: bsc#909589
Subject: [PATCH] RDMA/cxgb4: Increase epd buff size for debug interface

IPv6 address string lengths require increasing the buffer size for
debugfs handlers.

Signed-off-by: Pramod Kumar <pramod@chelsio.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
Acked-by: John Jolly <jjolly@suse.de>
---
 drivers/infiniband/hw/cxgb4/device.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/infiniband/hw/cxgb4/device.c
+++ b/drivers/infiniband/hw/cxgb4/device.c
@@ -566,7 +566,7 @@ static int ep_open(struct inode *inode,
 	idr_for_each(&epd->devp->stid_idr, count_idrs, &count);
 	spin_unlock_irq(&epd->devp->lock);
 
-	epd->bufsize = count * 160;
+	epd->bufsize = count * 240;
 	epd->buf = vmalloc(epd->bufsize);
 	if (!epd->buf) {
 		ret = -ENOMEM;
